src/Plugin/formatter/FormatterJsonApi.php

```diff
@@ -251,4 +252,32 @@ protected function getRequest() {
     return restful()->getRequest();
   }
 
+  /**
+   * Move the embedded resources to the included key.
```

Comment:
so we always want an `included` key for embeds, also for the simple json formatter?

I'm actually wondering if we need/want to keep the simple json, and default to json api? (out of scope of this PR of course)